label_studio/ml_models/models.py

```diff
@@ -126,8 +126,11 @@ class ModelRunStatus(models.TextChoices):
     )
 
     project_subset = models.CharField(max_length=255, choices=ProjectSubset.choices, default=ProjectSubset.HASGT)
+
     status = models.CharField(max_length=255, choices=ModelRunStatus.choices, default=ModelRunStatus.PENDING)
 
+    job_id = models.CharField(max_length=255, null=True, blank=True, default=None)
```

Comment:
nit: may be we should mention this is adala job id

Reply:
Currently it is yes, might not be in the future but I will add help text